conans/tools.py

```diff
@@ -22,7 +22,8 @@
 from conans.errors import ConanException
 from conans.model.version import Version
 # noinspection PyUnresolvedReferences
-from conans.util.files import _generic_algorithm_sum, load, save, sha256sum, sha1sum, md5sum, md5
+from conans.util.files import _generic_algorithm_sum, load, save, sha256sum, \
+    sha1sum, md5sum, md5, touch, relative_dirs, rmdir, mkdir, path_exists
```

Comment:
I think, `relative_dirs` might be handy for e.g. copying all license files out of a directory tree. Though I am with you with `path_exists`, now that I worked on the documentation. It seems a bit too specialized.